src/components/SimpleSlider/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import Slider from "react-slick";
+
+class SimpleSlider extends Component {
```

Comment:
Esto puede ser un componente puro. No tenemos constructor ni tenemos funciones dentro de este componente.

```suggestion
const SimpleSlider = (props) => {

const settings = {...}

return(
 ...
)
```